caches.py

```diff
@@ -6,12 +6,17 @@
 """
 import abc
 import errors
-import httplib
 import json
 import logger
 import os
 import threading as t
 
+import sys
+if sys.version_info[0] == 3:
+    import http.client as httplib
+else:
+    import httplib
```

Comment:
i'd prefer is we use the try/except import here, 1 less thing to import and we get away from magic values.

ie
```
try:
    import http.client as httplib
except ImportError:
    import httplib
```

Reply:
@elmiko good point! This was changed in https://github.com/radanalyticsio/jiminy-predictor/pull/34/commits/2249d109dfe1b2cd3e22cbc84492558c57454804